src/Handlebars/Context.php

```diff
@@ -73,6 +86,15 @@ public function pushIndex($index)
         array_push($this->index, $index);
     }
 
+    /**
+     * Pushes data variables onto the stack. This is used to support @data variables.
+     * @param array $data Associative array where key is the name of the @data variable and value is the value.
+     */
+    public function pushData($data)
+    {
+        array_push($this->dataStack, $data);
```

Comment:
If this is called and data variables are not enabled should an exception be thrown?

Reply:
Agreed and fixed.